### PR TITLE
Blog post with Luigi updates

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- Write a Luigi blog post explaining web components, new Luigi features and releases.

**Related issue(s)**
See also #1872

### CODE REVIEW

File: blog/2021-02-08-luigi-updates.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - Aleksandra Simeonova
+layout: blog
+---
+
+In this blog post, you can read about the most important updates and new features in the recent Luigi releases.
+<!-- Excerpt -->
+
+#### Web components
+
+In Luigi v1.7.0, we introduced a great new feature - support for Web Components. This has the potential to add a lot of new capablities to your Luigi apps, including a simple way to include more than one micro frontend on one page. You can find more resources and tutorials about Web Components [here](https://developer.mozilla.org/en-US/docs/Web/Web_Components). You can find the documentation for web components on our website [here](https://docs.luigi-project.io/docs/web-component).
```
Comment (legteodav) [inline blog/2021-02-08-luigi-updates.md:14]:
```suggestion
In Luigi v1.7.0, we introduced a great new feature: support for Web Components. This has the potential to add a lot of new capablities to your Luigi apps, including a simple way to include more than one micro frontend on one page. You can find more resources and tutorials about Web Components [here](https://developer.mozilla.org/en-US/docs/Web/Web_Components). You can find the documentation for web components on our website [here](https://docs.luigi-project.io/docs/web-component).
```

**Review by legteodav — APPROVED** [review]

File: blog/2021-02-08-luigi-updates.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+#### Fundamental Library Styles - breaking changes
+
+With Luigi v1.9.0, the new v0.14.0 of Fundamental Library Styles were included. Its previous v0.12.0 led to having breaking changes in the Luigi left side navigation and the shellbar. All icons were moved out from pseudo-classes into a dedicated tag <i class="sap-icon sap-icon--{modifier}"></i> under the button. You can see all breaking changes of Fundamental Library Styles v0.12.0 here.
+
+#### Confirmation modals
+
+With v.1.10.0, we introduced support for different types of confirmation modals. Modals can be of type `confirmation`, `success`, `error`, `warning`, or `information` and they can be configured using the [Luigi Core API](https://docs.luigi-project.io/docs/luigi-core-api/?section=showconfirmationmodal). The modal can also be hidden with the `settings.buttonConfirm` parameter.
```
Comment (stanleychh) [inline blog/2021-02-08-luigi-updates.md:28]:
```suggestion
With v.1.10.0, we introduced support for different types of confirmation modals. Modals can be of type `confirmation`, `success`, `error`, `warning`, or `information` and they can be configured using the [Luigi Core API](https://docs.luigi-project.io/docs/luigi-core-api/?section=showconfirmationmodal). The confirmation button can also be hidden with the `settings.buttonConfirm` parameter.
```

Comment (stanleychh) [reply blog/2021-02-08-luigi-updates.md:28]: It's not the whole modal, only the confirmation button. :)

**Review by stanleychh — APPROVED** [review]

File: blog/2021-02-08-luigi-updates.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+An example of Luigi Web Components can be found on our sandbox application [Luigi Fiddle](https://fiddle.luigi-project.io/). You can click the "Modify Config" button to see how the Web Components are configured in Luigi. The full source code of Luigi Fiddle can also be found on our [GitHub repo](https://github.com/SAP/luigi/tree/master/website/fiddle), in case you also want to delve deeper into the example.
+
+#### Luigi Client Angular support library
+
+Also in v.1.7.0, we introduced a support library for Luigi Client in Angular. After you import the library to your project, you can use different features related to routing in Luigi, as well as  You can find more information about the library [here](https://github.com/SAP/luigi/tree/master/client-frameworks-support/client-support-angular).
```
Comment (hardl) [inline blog/2021-02-08-luigi-updates.md:20]: something missing after "as well as"?

File: docs/web-component.md [hunk 4]
```diff
@@ -20,9 +20,9 @@ meta -->
 
 ### Overview
 
-Luigi offers the possibility to open a micro frontend as a web component; if you want to have more information about web component, please have a look at the page: [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
+Luigi offers the possibility to open a micro frontend as a web component; if you want to have more information about web components, please have a look at the page: [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components).
 
-For no-complex micro frontend, it can be a good and fast alternative: all the frontend will be loaded in a single javascript file.
+For non-complex micro frontend, it can be a good and fast alternative: all the frontend will be loaded in a single javascript file.
```
Comment (hardl) [inline docs/web-component.md:25]: "all the frontend" sounds a bit strange to me, and it is not relevant here imho.
you should rather mention, that it is for trusted sources only.

**Review by hardl — CHANGES_REQUESTED** [review]

File: docs/web-component.md (hunk 4 above)
Comment (alexandra-simeonova) [reply docs/web-component.md:25]: Hi, I'm not sure why this change is here in the blog post PR, I intended to open a new PR with a review of the web components doc. So I will try to remove this and add all additional changes in a new PR.

**Review by hardl — APPROVED** [review]

**Pull request merged**